### PR TITLE
Pgbench

### PR DESCRIPTION
**Pull request by dustinblack**
Submitting without CI for now; needs manual review until we have the postgres role in PR #107 merged

### CODE REVIEW

File: docs/pgbench.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# pgbench
+
+[pgbench](https://www.postgresql.org/docs/10/pgbench.html) is a performance test kit for PostgreSQL.
+
+## Running pgbench
+
+Given that you followed instructions to deploy operator,
+you can modify [cr.yaml](../resources/crds/ripsaw_v1alpha1_pgbench_cr.yaml) to your needs.
```
Comment (jtaleric) [inline docs/pgbench.md]: Throw the full name in here :

`s/cr.yaml/ripsaw_v1alpha1_pgbench_cr.yaml`

Comment (dustinblack) [reply docs/pgbench.md]: This is actually consistent across other docs.

Comment (jtaleric) [reply docs/pgbench.md]: yeah, I am systematically trying to get each updated...

Comment (dustinblack) [reply docs/pgbench.md]: Not sure how much I care about it, but it _will_ be harder to cleanup later for PR #104

Comment (jtaleric) [reply docs/pgbench.md]: It is mostly cosmetic.

**Comment by aakarshg** [comment]
CI passed on pgbench so merging :)

**Pull request merged**